src/encoded/tests/test_types_biosample.py

```diff
@@ -64,12 +64,12 @@ def test_update_biosample_relation(testapp, human_biosample, biosample_1):
 def test_biosample_calculated_properties(testapp, biosample_1, ):
     """
     Test to ensure the calculated properties are in result returned from testapp
+    These have string 'None' returned if no value as they are used in Item page view
     """
     res = testapp.get(biosample_1['@id']).json
     assert 'modifications_summary' in res
     assert 'modifications_summary_short' in res
     assert 'treatments_summary' in res
-    assert 'biosource_summary' in res
```

Comment:
Why is this line deleted? Is the prop gone or is the behavior different?

Reply:
An oversight - I was going to wrap that calcprop up with all others that used biosource or cell culture details info but Will recommended against it and I forgot to add this back here.